pkgs/by-name/gi/git-of-theseus/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "git-of-theseus";
+  version = "0.3.4";
+
+  # Source is not published to PyPI, have to use fetchFromGitHub.
+  # Also, version tags are inconsistently used, so we have to use a commit hash.
+  # i.e. https://github.com/erikbern/git-of-theseus/commit/1d77f082a9b25fb3a0c541641722cd4836135362#commitcomment-141438900
+  src = fetchFromGitHub {
+    owner = "erikbern";
+    repo = pname;
```

Comment:
use of pname like this is an anti-pattern and breaks overrides

NIT: Also, the comment about PyPI isn't necessary as fetchFromGitHub is preferred anyways